docs/users/index.md

```diff
@@ -25,15 +25,15 @@ BetterDiscord fully integrates Twitch emotes sourced from BTTV, Twitch, and FFZ.
 
 ### Live Editor
 
-BetterDiscord allows you to modify your Discord application and see the changes apply in real time.
+BetterDiscord allows you to modify your Discord application and see the changes apply in real-time.
```

Comment:
"Real time" is not being used as an adjective here and should not be hyphenated https://strategiesforparents.com/real-time-or-real-time-when-to-use-a-hyphen-and-when-to-skip-it/

Reply:
fixed.